tests/test_project.py

```diff
@@ -6,7 +6,7 @@ class TestProject():
 
     def test_project(self):
         proj = pipenv.project.Project()
-        assert proj.name == 'pipenv'
+        assert proj.name == 'pipenv-uoEO'
```

Comment:
There's no rush here, @simnalamburt :) I probably won't have a moment to respond to the other comments for a few hours, so don't feel like you need to stay up working on this.